internal/services/containers/kubernetes_flux_configuration_resource_test.go

```diff
@@ -715,3 +753,43 @@ resource "azurerm_kubernetes_flux_configuration" "test" {
 }
 `, template, data.RandomInteger)
 }
+
+func (r KubernetesFluxConfigurationResource) kustomizationPostBuild(data acceptance.TestData) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+				%s
+
+resource "azurerm_kubernetes_flux_configuration" "test" {
+  name       = "acctest-fc-%d"
+  cluster_id = azurerm_kubernetes_cluster.test.id
+  namespace  = "flux"
+
+  git_repository {
+    url             = "https://github.com/Azure/arc-k8s-demo"
+    reference_type  = "branch"
+    reference_value = "main"
+  }
+
+  kustomizations {
+    name = "kustomization-1"
+    path = "./test/path"
+    postBuild {
+      substitute = {
+        "key"  = "value",
+        "key2" = "value2"
+      }
+      substituteFrom {
```

Comment:
I think this should be 
```suggestion
    post_build {
      substitute = {
        "key"  = "value",
        "key2" = "value2"
      }
      substitute_from {
```

as we have a test failure:
```
------- Stdout: -------
=== RUN   TestAccKubernetesFluxConfiguration_kustomizationPostBuild
=== PAUSE TestAccKubernetesFluxConfiguration_kustomizationPostBuild
=== CONT  TestAccKubernetesFluxConfiguration_kustomizationPostBuild
    testcase.go:113: Step 1/2 error: Error running pre-apply refresh: exit status 1
        
        Error: Unsupported block type
        
          on terraform_plugin_test.tf line 75, in resource "azurerm_kubernetes_flux_configuration" "test":
          75:     postBuild {
        
        Blocks of type "postBuild" are not expected here. Did you mean "post_build"?
--- FAIL: TestAccKubernetesFluxConfiguration_kustomizationPostBuild (3.25s)
FAIL
```

Reply:
Fixed this in latest commit